plugins/importexport/native/PKPNativeImportExportPlugin.inc.php

```diff
@@ -0,0 +1,351 @@
+					__('plugins.importexport.native.export.submissions.results'),
+						'exportSubmissions',
+						array('selectedSubmissions' => $request->getUserVar('selectedSubmissions'))
+				);
+
+				$this->result = $tab;
+				$this->isResultManaged = true;
+
+				break;
+			case 'import':
+				$temporaryFilePath = $this->getImportedFilePath($request->getUserVar('temporaryFileId'), $user);
+				list ($filter, $xmlString) = $this->getImportFilter($temporaryFilePath);
+				$result = $this->getImportTemplateResult($filter, $xmlString, $this->getDeployment(), $templateMgr);
+
+				$this->result = $result;
+				$this->isResultManaged = true;
+
+				break;
+			case 'exportSubmissions':
+				$submissionIds = (array) $request->getUserVar('selectedSubmissions');
+
+				$this->getExportSubmissionsDeployment($submissionIds, $this->_childDeployment);
+
+				$result = $this->getExportTemplateResult($this->getDeployment(), $templateMgr, 'submissions');
+
+				$this->result = $result;
+				$this->isResultManaged = true;
+
+				break;
+			case 'downloadExportFile':
+				$downloadPath = $request->getUserVar('downloadFilePath');
+				$this->downloadExportedFile($downloadPath);
+
+				$this->isResultManaged = true;
+
+				break;
+		}
+	}
+
+	/**
+	 * Get the XML for a set of submissions.
+	 * @param $submissionIds array Array of submission IDs
+	 * @param $context Context
+	 * @param $user User|null
+	 * @param $opts array
+	 * @return string XML contents representing the supplied submission IDs.
+	 */
+	function exportSubmissions($submissionIds, $context, $user, $opts = array()) {
+		$appSpecificDeployment = $this->getAppSpecificDeployment($context, null);
+		$this->setDeployment($appSpecificDeployment);
+
+		$this->getExportSubmissionsDeployment($submissionIds, $appSpecificDeployment, $opts);
+
+		return $this->exportResultXML($appSpecificDeployment);
+	}
+
+	/**
+	 * @copydoc PKPImportExportPlugin::usage
+	 */
+	function usage($scriptName) {
+		echo __('plugins.importexport.native.cliUsage', array(
+			'scriptName' => $scriptName,
+			'pluginName' => $this->getName()
+		)) . "\n";
+	}
+
+	/**
+	 * Load necessary locales for CLI
+	 */
+	function loadCLILocales() {
+		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_MANAGER, LOCALE_COMPONENT_PKP_SUBMISSION);
+	}
+
+	/**
+	 * @see PKPImportExportPlugin::executeCLI()
+	 */
+	function executeCLI($scriptName, &$args) {
+		$this->loadCLILocales();
+
+		$cliDeployment = new PKPNativeImportExportCLIDeployment($scriptName, $args);
+		$this->cliDeployment = $cliDeployment;
+
+		$contextDao = Application::getContextDAO(); /** @var $contextDao ContextDAO */
+		$userDao = DAORegistry::getDAO('UserDAO'); /** @var $userDao UserDAO */
+
+		$contextPath = $cliDeployment->contextPath;
+		$context = $contextDao->getByPath($contextPath);
+
+		if (!$context) {
+			if ($contextPath != '') {
+				$this->cliToolkit->echoCLIError(__('plugins.importexport.common.error.unknownJournal', array('contextPath' => $contextPath)));
```

Comment:
OJS-specific language needs changing (`unknownJournal`)

Reply:
Changed (also elsewhere) and key moved to pkp-lib 

`plugins.importexport.common.cliError` also moved to pkp-lib

I added the `manager.po` locale file for `hi_IN` and `sk_SK` because there was no file in pkp-lib.